src/main/java/pl/cinema/server/reservation/ReservationHandler.java

```diff
@@ -0,0 +1,37 @@
+package pl.cinema.server.reservation;
+
+
+import lombok.RequiredArgsConstructor;
+import pl.cinema.server.customer.CustomerFacade;
+import pl.cinema.server.customer.dto.CustomerDto;
+import pl.cinema.server.reservation.dto.ReservationCommand;
+import pl.cinema.server.reservation.dto.ReservationConfirmationDto;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RequiredArgsConstructor
+class ReservationHandler {
```

Comment:
w innych pakietach miałeś publiczne facade tu masz pakietowy handler. Dlaczego tak?

Reply:
nie potrzebuję póki co do wymagań z zadania fasady z tego pakirtu, w obrębie reservation działa ten handler, nie musi być publiczny, jakby inna domena chciała pobierać coś o rezerwacjach to bym zrobił Fasadę